Makefile

```diff
@@ -3,7 +3,7 @@ PIP := .env/bin/pip
 PYTHON := .env/bin/python
 TOX := .env/bin/tox
 STAGE := DEV
-HUMILIS_ENV := tests/integration/humilis-sam
+HUMILIS_ENV := tests/integration/humilis-sam-swagger tests/integration/humilis-sam-classic
```

Comment:
Minor fixes to makefile so that the `create`, `delete` and `update` targets work.